cmd/go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/containerd/typeurl/v2 v2.1.1 // indirect
 	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/docker v24.0.7+incompatible // indirect
```

Comment:
shouldn't this version match the dependency in docker/cli?

Reply:
Since we upgraded cli to 25.0.5 https://github.com/awslabs/soci-snapshotter/pull/1126, it depends on 25.0.5 of docker https://github.com/docker/cli/blob/v25.0.5/vendor.mod#L15

Reply:
@swagatbora90 pushed an update to push docker/docker dependency to 25.0.5.

Also just so it is clear we don't use much of docker/docker. Only https://github.com/moby/moby/tree/25.0/pkg/homedir is used.